ssl/ktls.c

```diff
@@ -131,8 +131,7 @@ int ktls_check_supported_cipher(const SSL *s, const EVP_CIPHER *c,
      */
 # ifdef OPENSSL_KTLS_AES_CCM_128
     if (EVP_CIPHER_is_a(c, "AES-128-CCM")) {
-        if (s->version == TLS_1_3_VERSION /* broken on 5.x kernels */
-            || EVP_CIPHER_CTX_get_tag_length(dd) != EVP_CCM_TLS_TAG_LEN)
+        if (EVP_CIPHER_CTX_get_tag_length(dd) != EVP_CCM_TLS_TAG_LEN)
```

Comment:
how can you be sure that **this** kernel is no a broken one?

Reply:
The problem is the kernel on the build system can be even completely different version from the one where the code will later run.